uelc/settings_shared.py

```diff
@@ -20,7 +20,7 @@
     }
 }
 
-if 'test' in sys.argv or 'jenkins' in sys.argv:
+if 'jenkins' in sys.argv:
```

Comment:
So that the tests run on postgresql instead of sqlite when running `./manage.py test`. I did this because sqlite wasn't raising an error that occurs in postgresql when the username is greater than 30 characters.

I'll change this to add a `settings_integration.py` like dmt has to make this more explicit (https://github.com/ccnmtl/dmt/blob/master/dmt/settings_integration.py).